'No space left on device' errors by Neo4j and Postgres

Running the benchmark on neo and pos fails when the current user has limited write permissions to system directories.

### Postgres

Starting the container (without detaching) fails with:
```
fixing permissions on existing directory /var/lib/postgresql/data ... ok
initdb: error: could not create directory "/var/lib/postgresql/data/pg_wal": No space left on device
initdb: removing contents of data directory "/var/lib/postgresql/data"
```

### Neo4j

Starting the container works, but loading a dataset throws Exceptions such as:
```
org.neo4j.logging.shaded.log4j.core.appender.AppenderLoggingException: Error writing to stream /logs/debug.log
        at org.neo4j.logging.shaded.log4j.core.appender.OutputStreamManager.writeToDestination(OutputStreamManager.java:252)
        at org.neo4j.logging.shaded.log4j.core.appender.FileManager.writeToDestination(FileManager.java:277)
[...]
Caused by: java.io.IOException: No space left on device
        at java.base/java.io.FileOutputStream.writeBytes(Native Method)
        at java.base/java.io.FileOutputStream.write(FileOutputStream.java:354)
```

I found a fix that I will share shortly.

--- a/neo/load.sh
+++ b/neo/load.sh
@@ -22,6 +22,7 @@ docker run \
     --publish=7474:7474 \
     --publish=7687:7687 \
     --volume=${NEO4J_DATA_DIR}:/data:z \
+    --volume=${NEO4J_HOME}/logs:/logs:z \
     --volume=${IMPORT_DATA_DIR_PROJECTED_FK}:/import:z \
     ${NEO4J_ENV_VARS} \
     neo4j:${NEO4J_VERSION} \

--- a/pos/scratch/.gitignore
+++ b/pos/scratch/.gitignore
@@ -1,0 +1,3 @@
+*
+!.gitignore
+

--- a/pos/start.sh
+++ b/pos/start.sh
@@ -15,6 +15,7 @@ docker run \
     --name ${POSTGRES_CONTAINER_NAME} \
     --env ${POSTGRES_PASSWORD_POLICY} \
     --volume=${IMPORT_DATA_DIR_MERGED_FK}:/data:z \
+	--volume=${POSTGRES_HOME}:/var/lib/postgresql/data:z \
     --detach \
     --shm-size=${POSTGRES_SHARED_MEMORY} \
     postgres:${POSTGRES_VERSION}

--- a/pos/vars.sh
+++ b/pos/vars.sh
@@ -1,3 +1,4 @@
+export POSTGRES_HOME=`pwd`/pos/scratch
 export POSTGRES_VERSION=13.1
 export POSTGRES_SCHEMA_DIR=`pwd`/sql
 export POSTGRES_CONTAINER_NAME=lsqb-pos

--- a/sql/drop.sql
+++ b/sql/drop.sql
@@ -10,6 +10,8 @@ DROP TABLE IF exists University;
 DROP TABLE IF exists Continent;
 DROP TABLE IF exists Country;
 DROP TABLE IF exists City;
+DROP TABLE IF exists Tag;
+DROP TABLE IF exists TagClass;
 DROP TABLE IF exists Forum;
 DROP TABLE IF exists Comment;
 DROP TABLE IF exists Post;

